probe_src/python/probe_py/manual/cli.py

```diff
@@ -145,7 +151,12 @@ def dataflow_graph(
     process_graph = analysis.provlog_to_digraph(prov_log)
     for warning in analysis.validate_hb_graph(prov_log, process_graph):
         console.print(warning, style="red")
-    print(analysis.provlog_to_dataflow_graph(prov_log))
+
+    dot_string, dataflow_graph = analysis.provlog_to_dataflow_graph(prov_log)
+    print(dot_string)
+    import pickle
+    with open(output, 'wb') as f:
+        pickle.dump(dataflow_graph, f)
 
```

Comment:
I needed to do pickle/unpickle to send you the graph before `provlog_to_digraph` was implemented (I think).

Now we should simply call `analysis.provlog_to_dataflow_graph(prov_log)` in the `export`.

This way, there is only one file users need to store, the `probe_log`, and the rest is generated based on that on-the-fly, since those transformations are (hopefully) cheap.